python/cugraph/components/cc_wrapper.pyx

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c_cc cimport *
+#from libcpp cimport bool
+from c_graph cimport *
+#from libcpp cimport cugraph_cc_t
+from libc.stdint cimport uintptr_t
+import cudf
+#from pygdf import Column
+import numpy as np
+
+cpdef weak_cc(G, connect_type = CUGRAPH_WEAK):
+    """
+    Find the distances and predecessors for a breadth first traversal of a graph.
+    
+    Parameters
+    ----------
+    G : cugraph.graph
+        cuGraph graph descriptor, should contain the connectivity information as an
```

Comment:
Doc should say more about what input are supported. supported input. Does the graph need to be undirected for wcc? If an undirected edge is represented by an edge in both direction it should also be mentioned here. Indexes must be 32b I suppose. etc.

Reply:
There has been extensive talk about being able to call WeakCC on directed graphs in the future. In that venue, the API shouldn't forbid (in the future) calling directed graphs.

Reply:
I agree but the documentation does not reflect this